.ci/cico_build_and_deploy_custom_rh-che.sh

```diff
@@ -5,38 +5,33 @@
 # which accompanies this distribution, and is available at
 # http://www.eclipse.org/legal/epl-v10.html
 
+# ERROR CODES:
+# 1 - missing file
+# 2 - missing credentials
+# 3 - OpenShift login failed
+# 4 - command execution failed
+
 set -x
 set -e
 set +o nounset
 
 /usr/sbin/setenforce 0
 
-export CHE_DOCKER_BASE_IMAGE=eclipse/che-server:nightly-centos
-export ORIGIN_CLIENTS_URL=http://mirror.centos.org/centos/7/paas/x86_64/openshift-origin/origin-clients-3.7.1-2.el7.x86_64.rpm
-export OPENSHIFT_RDU2C_URL=https://dev.rdu2c.fabric8.io:8443/
-export OC_VERSION=3.9.19
-
-function tagAndPushDocker() {
-  export RH_CHE_AUTOMATION_BUILD_TAG=rh-che-automated-build-$(git rev-parse --short HEAD)
-  echo "che-docker-base-image:${CHE_DOCKER_BASE_IMAGE}"
-  docker pull ${CHE_DOCKER_BASE_IMAGE}
-  docker tag ${CHE_DOCKER_BASE_IMAGE} eclipse/che-server:local
-  rm -rf ./dockerfiles/che-fabric8/eclipse-che
-  mkdir ./dockerfiles/che-fabric8/eclipse-che
-  cp -r ./assembly/assembly-main/target/eclipse-che-fabric8-1.0.0-SNAPSHOT/eclipse-che-fabric8-1.0.0-SNAPSHOT/* ./dockerfiles/che-fabric8/eclipse-che/
-  bash ./dockerfiles/che-fabric8/build.sh || exit 1
-  docker rmi eclipse/che-server:local
-  docker rmi ${CHE_DOCKER_BASE_IMAGE}
-  docker tag eclipse/che-server:nightly rhcheautomation/che-server:${RH_CHE_AUTOMATION_BUILD_TAG}
-  docker rmi eclipse/che-server:nightly
-  set +x
-  docker login -u ${RH_CHE_AUTOMATION_DOCKERHUB_USERNAME} -p ${RH_CHE_AUTOMATION_DOCKERHUB_PASSWORD}
-  set -x
-  docker push rhcheautomation/che-server:${RH_CHE_AUTOMATION_BUILD_TAG}
+export RH_CHE_AUTOMATION_SERVER_DEPLOYMENT_URL=https://rhche-che6-automated.dev.rdu2c.fabric8.io/
+export BASEDIR=$(pwd)
+export DEV_CLUSTER_URL=https://dev.rdu2c.fabric8.io:8443/
+export OC_VERSION=3.7.48
+export TARGET="rh-integration-test"
+
+function BuildTagAndPushDocker() {
+  echo "Docker status:"
+  docker images
+  DeveloperBuild="true" .ci/cico_build.sh
```

Comment:
`DeveloperBuild` was ment to set to `true` when running locally on a dev laptop. But here, for PR check purposes you are setting it to true for cento-ci. And for build of the Docker images you set it to false. That is extremely confusing and really easy to break. Can't you just use a new variable `PRCheckBuild` and set it to true here and then checking it in `cico_build.sh`?